src/components/Box/BoxConstants.js

```diff
@@ -0,0 +1,262 @@
+  XS: styles.meXs,
+  SMALL: styles.meSmall,
+  MEDIUM: styles.meMedium,
+  LARGE: styles.meLarge,
+  XL: styles.meXl,
+  XXL: styles.meXxl,
+  XXXL: styles.meXxxl
+});
+
+export const MARGIN_BOTTOM = Object.freeze({
+  NONE: styles.mb0,
+  AUTO: styles.mbAuto,
+  XS: styles.mbXs,
+  SMALL: styles.mbSmall,
+  MEDIUM: styles.mbMedium,
+  LARGE: styles.mbLarge,
+  XL: styles.mbXl,
+  XXL: styles.mbXxl,
+  XXXL: styles.mbXxxl
+});
+
+export const MARGIN_START = Object.freeze({
+  NONE: styles.ms0,
+  AUTO: styles.msAuto,
+  XS: styles.msXs,
+  SMALL: styles.msSmall,
+  MEDIUM: styles.msMedium,
+  LARGE: styles.msLarge,
+  XL: styles.msXl,
+  XXL: styles.msXxl,
+  XXXL: styles.msXxxl
+});
+
+export const PADDING = Object.freeze({
+  NONE: styles.p0,
+  XS: styles.pXs,
+  SMALL: styles.pSmall,
+  MEDIUM: styles.pMedium,
+  LARGE: styles.pLarge,
+  XL: styles.pXl,
+  XXL: styles.pXxl,
+  XXXL: styles.pXxxl
+});
+
+export const PADDING_X = Object.freeze({
+  NONE: styles.px0,
+  XS: styles.pxXs,
+  SMALL: styles.pxSmall,
+  MEDIUM: styles.pxMedium,
+  LARGE: styles.pxLarge,
+  XL: styles.pxXl,
+  XXL: styles.pxXxl,
+  XXXL: styles.pxXxxl
+});
+
+export const PADDING_Y = Object.freeze({
+  NONE: styles.py0,
+  XS: styles.pyXs,
+  SMALL: styles.pySmall,
+  MEDIUM: styles.pyMedium,
+  LARGE: styles.pyLarge,
+  XL: styles.pyXl,
+  XXL: styles.pyXxl,
+  XXXL: styles.pyXxxl
+});
+
+export const PADDING_TOP = Object.freeze({
+  NONE: styles.pt0,
+  XS: styles.ptXs,
+  SMALL: styles.ptSmall,
+  MEDIUM: styles.ptMedium,
+  LARGE: styles.ptLarge,
+  XL: styles.ptXl,
+  XXL: styles.ptXxl,
+  XXXL: styles.ptXxxl
+});
+
+export const PADDING_END = Object.freeze({
+  NONE: styles.pe0,
+  XS: styles.peXs,
+  SMALL: styles.peSmall,
+  MEDIUM: styles.peMedium,
+  LARGE: styles.peLarge,
+  XL: styles.peXl,
+  XXL: styles.peXxl,
+  XXXL: styles.peXxxl
+});
+
+export const PADDING_BOTTOM = Object.freeze({
+  NONE: styles.pb0,
+  XS: styles.pbXs,
+  SMALL: styles.pbSmall,
+  MEDIUM: styles.pbMedium,
+  LARGE: styles.pbLarge,
+  XL: styles.pbXl,
+  XXL: styles.pbXxl,
+  XXXL: styles.pbXxxl
+});
+
+export const PADDING_START = Object.freeze({
+  NONE: styles.ps0,
+  XS: styles.psXs,
+  SMALL: styles.psSmall,
+  MEDIUM: styles.psMedium,
+  LARGE: styles.psLarge,
+  XL: styles.psXl,
+  XXL: styles.psXxl,
+  XXXL: styles.psXxxl
+});
+
+export const BACKGROUND_COLORS = Object.freeze({
+  PRIMARY_BACKGROUND_COLOR: styles.bgPrimaryBackgroundColor,
+  PRIMARY_HOVER_BACKGROUND_COLOR: styles.bgPrimaryBackgroundHoverColor,
+  SECONDARY_BACKGROUND_COLOR: styles.bgSecondaryBackgroundColor,
+  GREY_BACKGROUND_COLOR: styles.bgGreyBackgroundColor,
+  ALL_GREY_BACKGROUND_COLOR: styles.bgAllgreyBackgroundColor,
+  INVERTED_COLOR_BACKGROUND: styles.bgInvertedColorBackground,
+  DISABLED_BACKGROUND_COLOR: styles.bgDisabledBackgroundColor
+});
+
+export const COLORS = Object.freeze({
+  PRIMARY_COLOR: styles.textPrimaryColor,
+  PRIMARY_HOVER_COLOR: styles.textPrimaryHoverColor,
```

Comment:
for the following colors: hoverColor, selectedColor, iconColor,LinkColor, placeholderColor - IMO they shouldn't be an option to use them in a container - they are either too component specific or they are shouldn't be used

Reply:
+1